common/src/main/java/us/myles/ViaVersion/protocols/protocol1_13to1_12_2/Protocol1_13To1_12_2.java

```diff
@@ -189,18 +188,22 @@ public void registerMap() {
                 handler(new PacketHandler() {
                     @Override
                     public void handle(PacketWrapper wrapper) throws Exception {
-                        int size = wrapper.passthrough(Type.VAR_INT);
+                        int size = wrapper.read(Type.VAR_INT);
+                        List<Triple<Integer, Integer, Integer>> remappedStats = new ArrayList<>();
```

Comment:
Create a new class in the data package to put the 3 ints into instead of using a triple (to avoid frequent primitive wrapping)